src/installation/musl.md

```diff
@@ -48,9 +48,9 @@ Use [chroot(1)](https://man.voidlinux.org/chroot.1) to change to the new root,
 then run glibc programs as usual. Once you've finished using it, unmount the
 chroot using [umount(8)](https://man.voidlinux.org/umount.8).
 
-#### PRoot
+#### bubblewrap
 
-An alternative to the above is [proot(1)](https://man.voidlinux.org/proot.1), a
-user-space implementation of chroot, mount --bind, and binfmt_misc. By
-installing the `proot` package, unprivileged users can utilize a chroot
-environment.
+An alternative for unprivileged users is the
+[bwrap(1)](https://man.voidlinux.org/bwrap) command which uses Linux namespaces
+to run applications in a sandbox container. It can be installed via the
+`bubblewrap` package.
```

Comment:
```suggestion
An alternative for unprivileged users is
[bwrap(1)](https://man.voidlinux.org/bwrap.1), which uses Linux namespaces. It can be installed via the
`bubblewrap` package. An example invocation is shown below:
```

Then include an example of how to use it, since it's not always obvious, IMO.

Reply:
I omitted an example invocation because from [CONTRIBUTING](https://github.com/void-linux/void-docs/blob/master/CONTRIBUTING.md) I inferred that it's preferred that readers study the manual page and decide for themselves which options will best suit their needs. Anyway, if you have any experience with this utility, please propose some full example, as all I needed to do in my case was
```bash
$ bwrap --bind path/to/chroot / command
```
which seems too basic.

Reply:
I think in this case the command is helpful, since what's desired from the tool isn't immediately obvious. I'd suggest `--proc /proc` and `--dev /dev`, at least.

Reply:
Actually, I'm currently unable to get a container set up using
```bash
$ XBPS_TARGET_ARCH=x86_64 xbps-install -S -r /tmp/chroot -R https://... base-voidstrap
```
to work properly - I have used
```bash
$ bwrap --bind /tmp/chroot / --uid 0 --gid 0 --dev  /dev --proc /proc --ro-bind /etc/resolv.conf /etc/resolv.conf sh
```
to chroot into it, but when I try to `ping voidlinux.org`, I get `ping: socket: Operation not permitted`. What am I missing?

Reply:
Nevermind, apparently it's because of capabilities.